addons/crm/static/src/views/crm_kanban/crm_kanban_model.js

```diff
@@ -45,7 +44,7 @@ export class CrmKanbanDynamicGroupList extends CrmKanbanModel.DynamicGroupList {
             sourceGroup.groupByField.name === "stage_id"
         ) {
             const record = targetGroup.list.records.find((r) => r.id === dataRecordId);
-            await checkRainbowmanMessage(this.model.ormService, this.model.effect, record.resId);
+            await checkRainbowmanMessage(this.model.orm, this.model.effect, record.resId);
```

Comment:
same

Reply:
As discussed, since `CrmKanbanDynamicGroupList` extends `DataPoint` initially and not `component`, keeping `model.orm` and `model.effect` here